test/entry_parser_test.ts

```diff
@@ -61,6 +61,14 @@ describe("EntryParser", () => {
       expect(parsedAuthor.uri).to.equals("test uri");
     });
 
+    it("extracts subtitle", () => {
+      let entry = {
+        "schema:alternativeHeadline": [{"_": "test subtitle"}]
```

Comment:
Just curious, only the first object is used? Any more are ignored but that's okay?

Reply:
Yeah. I guess multiple objects would mean the OPDS entry has multiple subtitles, which might be valid OPDS but doesn't make sense, so I think it would be safe to use the first one.